api/app/controllers/mno_enterprise/impersonate_controller.rb

```diff
@@ -39,6 +40,10 @@ def destroy
 
     private
 
+    def skip_trackable
```

Comment:
This part is not needed, the function is already defined in the `ApplicationController`

Reply:
Ah I see. 

I'm a little confused as to why this is in the application_controller. Trackables are only updated when the user signs in, correct? I see several `before_filter :authenticate_user!`, does it have to do with that?

Reply:
Sorry this was added in #578 and hasn't made to the `4.0` branch yet.
The method is in the `ApplicationController` so it can be called from any controller that might requires it.

Added more details in JIRA